srcpkgs/xdg-desktop-portal-gnome/template

```diff
@@ -14,7 +14,7 @@ short_desc="GNOME portal backend for xdg-desktop-portal"
 maintainer="oreo639 <oreo6391@gmail.com>"
 license="LGPL-2.1-or-later"
 homepage="https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome"
-changelog="https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/raw/gnome-45/NEWS"
 #changelog="https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/raw/main/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/raw/46.0/NEWS"
```

Comment:
This should be `https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/raw/gnome-46/NEWS`